modules/core/shared/src/main/scala/gem/enum/F2ReadMode.scala

```diff
@@ -18,18 +18,18 @@ sealed abstract class F2ReadMode(
   val shortName: String,
   val longName: String,
   val description: String,
-  val minimumExposureTime: Duration,
-  val recommendedExposureTime: Duration,
-  val readoutTime: Duration,
+  val minimumExposureTime: FiniteDuration,
+  val recommendedExposureTime: FiniteDuration,
+  val readoutTime: FiniteDuration,
   val readCount: Int,
   val readNoise: Double
 ) extends Product with Serializable
 
 object F2ReadMode {
 
-  /** @group Constructors */ case object Bright extends F2ReadMode("Bright", "bright", "Bright Object", "Strong Source", Duration.ofMillis(1500), Duration.ofMillis(5000), Duration.ofMillis(8000), 1, 11.7)
-  /** @group Constructors */ case object Medium extends F2ReadMode("Medium", "medium", "Medium Object", "Medium Source", Duration.ofMillis(6000), Duration.ofMillis(21000), Duration.ofMillis(14000), 4, 6.0)
-  /** @group Constructors */ case object Faint extends F2ReadMode("Faint", "faint", "Faint Object", "Weak Source", Duration.ofMillis(12000), Duration.ofMillis(85000), Duration.ofMillis(20000), 8, 5.0)
+  /** @group Constructors */ case object Bright extends F2ReadMode("Bright", "bright", "Bright Object", "Strong Source", new FiniteDuration(1500, java.util.concurrent.TimeUnit.MILLISECONDS), new FiniteDuration(5000, java.util.concurrent.TimeUnit.MILLISECONDS), new FiniteDuration(8000, java.util.concurrent.TimeUnit.MILLISECONDS), 1, 11.7)
```

Comment:
In `EnumDef` maybe we can change the import corresponding to `FiniteDuration` into `Some("scala.concurrent.duration._")` and then we can write these as `1500 millis` instead of `new FiniteDuration(1500, java.util.concurrent.TimeUnit.MILLISECONDS)`.  I think it would clean up the enumerated code quite a bit.